tests/suites/test_suite_ssl.function

```diff
@@ -3747,14 +3744,13 @@ void handshake_version( int version, int dtls )
 
     options.version = version;
     options.dtls = dtls;
-    /* Note - the case below will have to updated, since the test sends no data
-     * due to a 1n-1 split against BEAST, that was not expected when preparing
-     * the fragment counting code. */
+    /* By default, SSLv3.0 and TLSv1.0 use 1/n-1 splitting when sending data, so
+     * the number of fragments will be twice bigger. */
```

Comment:
English suggestion:
```diff
- twice bigger.
+ twice as big.
```

Reply:
That got me thinking too! And it turns out that some people use it this way:
https://forum.thefreedictionary.com/postst5851_twice-bigger-.aspx

Reply:
Old fashioned, IMO